### ISSUE TITLE
Feature request: "/push" and "/pop" the loaded files; store the state in .aider.stack.md

### ISSUE THREAD

**Issue by jerzydziewierz** [paul-gauthier/aider#1619]
### Issue

I find myself frequently restarting aider with needing to load the same files using /add and /read e.t.c. ; 

may I suggest a new RPN-like commands to save/load the work-file-list from a local on-disk memory e.g. .aider.files.md

loading these from a config file is not applicable because this list will evolve as you progress with a project.

### Version and model info

Aider version: 0.56.0
Python version: 3.10.14
Platform: Linux-6.8.0-40-generic-x86_64-with-glibc2.35
Python implementation: CPython
Virtual environment: No
OS: Linux 6.8.0-40-generic (64bit)
Git version: git version 2.34.1

Aider v0.56.0
Main model: claude-3-5-sonnet-20240620 with diff edit format, infinite output
Weak model: claude-3-haiku-20240307
Git repo: .git with 97 files

**Comment by fry69** [comment]
Thank you for filing this issue.

An alternative way to specify files you need frequently is via command line ->
```shell
aider --read CONVENTIONS.md app/*.py
```
or via a `.aider.conf.yml` file in your repository root (though you cannot use shell globs (*.py) there):
```yaml
read: [CONVENTIONS.md, lib/util.py]
file: [app/frontend.py, app/server.py]
```
Either store these command lines in shell aliases or create different configuration files with different settings/files. You can specify a specific configuration file on the command line with `--config <file>`

This should give you options to preserve states/sets of files you regularly need and recall them on demand, if really needed. For even more advanced usage, have a look at the scripting support for aider:

https://aider.chat/docs/scripting.html

**Comment by jerzydziewierz** [comment]
Yes, well, I specifically want to avoid adding files from command line, because in a practical project, there are many of them, like 10+; and, the set changes over time

please see

https://github.com/paul-gauthier/aider/pull/1620

**Comment by paul-gauthier** [comment]
Thanks for trying aider and filing this issue.

FYI, aider has had `/load` and `/save` commands for awhile now.
